### PR TITLE
LPS-187960 Update usage of ModelListener.onAfterUpdate(T model) with ModelListener.onAfterUpdate(T originalModel, T model)

### PR DESCRIPTION
**Pull request by TamyrisTorres**
Ticket: https://liferay.atlassian.net/browse/LPS-187960

### CODE REVIEW

**Comment by TamyrisTorres** [comment]
ci:test:sf

**Comment by TamyrisTorres** [comment]
ci:test:relevant

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaOnAfterUpdateParameterCheck.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+		return StringUtil.replace(
+			newContent, methodCall,
+			StringUtil.replace(
+				methodCall, "onAfterUpdate(" + parameterList.get(0),
+				"onAfterUpdate(" + newParameter));
+	}
+
+	private String _toAssembleMethod(List<String> parameterList) {
+		String parameter = parameterList.get(0);
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaOnAfterUpdateParameterCheck.java:97]: @TamyrisTorres, let's rename to `firstParameter` instead

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaOnAfterUpdateParameterCheck.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+		int index = parameter.indexOf(" ") + 1;
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaOnAfterUpdateParameterCheck.java:99]: @TamyrisTorres, let's rename to `firstParameterNameIndex` instead

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaOnAfterUpdateParameterCheck.java [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+		String newParameter = _toAssembleMethod(parameterList);
+
+		return StringUtil.replace(
+			newContent, methodCall,
+			StringUtil.replace(
+				methodCall, "onAfterUpdate(" + parameterList.get(0),
+				"onAfterUpdate(" + newParameter));
+	}
+
+	private String _toAssembleMethod(List<String> parameterList) {
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaOnAfterUpdateParameterCheck.java:96]: @TamyrisTorres, let's rename to `newParameters` and `cloneAndRenameFirstParameter` instead, I mean:

`String newParameters = _cloneAndRenameFirstParameter(parameterList);`

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Comment by TamyrisTorres** [comment]
ci:test:relevant

**Review by natocesarrego — APPROVED** [review]
Good job! :smile:

**Comment by natocesarrego** [comment]
@TamyrisTorres, just to make sure we're on the same page, there is no expected action from my side regarding this pull request, am I right?

**Comment by TamyrisTorres** [comment]
@natocesarrego No. I already send it to Kevin. 
I Will Close this PR.

**Pull request closed without merging**